### ISSUE TITLE
Accessibility - marker/cluster reads out as clickable even when there is not a popup

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet-leaflet#63]
If the marker/cluster doesn’t have a popup the screenreader still considers it a clickable button even though clicking doesn’t do anything.
